src/PowerShell/ServicePrincipalLogin.ts

```diff
@@ -0,0 +1,39 @@
+import * as core from '@actions/core';
+
+import Utils from './Utilities/Utils';
+import PowerShellToolRunner from './Utilities/PowerShellToolRunner';
+import ScriptBuilder from './Utilities/ScriptBuilder';
+import Constants from './Constants';
+
+export class ServicePrincipalLogin implements IAzurePowerShellSession {
+    static readonly environment: string = Constants.environment;
```

Comment:
Rename constants to AzureCloud, Subscription, ServicePrincipal. Looking at the constant, we should understand the context.